lib/tic_tac_toe_elixir.ex

```diff
@@ -1,11 +1,46 @@
+defmodule Board do
+  def print_board(board_values) do
+    list = Regex.scan(~r/.../, get_board_string(board_values))
+    Enum.join(list, "\n")
+  end
+
+  def get_board_string(board_values) do
+    if board_values == nil do
```

Comment:
*thought:* as you progress further into a functional style, you'll find that sometimes you might want to break away from more 'imperative' control structures like if/else - this looks like 1) the identity function and 2) checking for `nil`.

You could perhaps look to break that into two smaller functions as needed, with a guard clause.

https://elixir-lang.org/getting-started/modules-and-functions.html#named-functions